### DISCUSSION TITLE
Github should also have multiple account logins in one portal.

### DISCUSSION THREAD

**Discussion by divyansh-mishra-07** [community/community#20364] [Other Feature Feedback, Questions, & Ideas]
I have three accounts to handle clients and personal projects but unfortunately, I am not able to access all of them at once. I have to use 3 separate browsers for each GitHub account. I request developers to think about it, please.

**Comment by NellyisDevv** [comment]
If you don't already do this, Google Chrome actually allows you to add more than one profile to your browser, so you would be able to just switch profiles by clicking the profile icon and switching instead of having to open an entire new browser page!

**Reply by NellyisDevv** [reply]
I believe Vivaldi has something similar and also allows for custom CSS to customize the browser maybe someone has already made some kind of feature

**Reply by smil2k** [reply]
> If you don't already do this, Google Chrome actually allows you to add more than one profile to your browser, so you would be able to just switch profiles by clicking the profile icon and switching instead of having to open an entire new browser page!

How? I have an enterprise and a personal account using both daily. My github experience is in the ashes since the enterprise user added.

**Comment by SpitFire-666** [comment]
Firefox Containers makes it easy to run multiple profiles in the one browser window. Edge and Chrome also let you have separate profiles, although they launch as separate, completely isolated instances.
